R/estimate_KM.R

```diff
@@ -146,8 +147,9 @@ estimate_KM <- function(
 
 
 # Update Call with original info and dots, similar as update.default ------
-  
-  survfit_object$call[[1]] <- quote(survival::survfit)
+
+  survfit_object[["estimate_KM_args"]] <- estimate_KM_args
```

Comment:
adding the entire list of information seems redundant. We have all this information already in the survfit object.

Reply:
updated to just return the strata label